frontend/src/features/contacts/contact/detail/Organization.tsx

```diff
@@ -0,0 +1,215 @@
+import { FormSection } from 'components/common/form/styles';
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { Dictionary } from 'interfaces/Dictionary';
+import {
+  IContactAddress,
+  IContactMethod,
+  IContactOrganization,
+  IContactPerson,
+} from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { FaCircle, FaRegBuilding } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+
+export interface OrganizationViewProps {
+  organization: IContactOrganization;
+}
+
+interface ContactInfoField {
+  info: string;
+  label: string;
+}
+
+interface AddressField {
+  address: IContactAddress;
+  label: string;
+}
+
+const OrganizationView: React.FunctionComponent<OrganizationViewProps> = ({ organization }) => {
+  const phoneTypes: Dictionary<string> = {};
```

Comment:
This is probably fine in this case, but be careful with this in stateful screens. All of this initialization will happen **every time** this component is re-rendered, which is an unnecessary use of resources. You can either memoize this work within a function component, or you can move this out of the component to prevent it from getting re-initialized unnecessarily.

Reply:
That is a good point. I believe in this case is fine since if the organization changes the whole components needs to be re-calculated